### PR TITLE
[ArticleBundle] Add deprecation error when using entities directly th…

### PR DESCRIPTION
**Pull request by treeleaf**
…at should be abstract

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | yes
| Fixed tickets | #1760

### CODE REVIEW

File: src/Kunstmaan/ArticleBundle/Entity/AbstractAuthor.php [hunk 1]
```diff
@@ -11,6 +11,16 @@
  */
 class AbstractAuthor extends AbstractEntity
 {
+    /**
+     * AbstractAuthor constructor.
+     */
+    public function __construct()
+    {
+        if (get_class($this) === AbstractAuthor::class) {
+            trigger_error('Please extend this class, it will be made abstract in 5.1.', E_USER_DEPRECATED);
```
Comment (Devolicious) [inline src/Kunstmaan/ArticleBundle/Entity/AbstractAuthor.php]: version 6.0, not 5.1

File: src/Kunstmaan/ArticleBundle/Entity/AbstractCategory.php [hunk 2]
```diff
@@ -10,6 +10,16 @@
 
 class AbstractCategory extends AbstractEntity implements Translatable
 {
+    /**
+     * AbstractCategory constructor.
+     */
+    public function __construct()
+    {
+        if (get_class($this) === AbstractCategory::class) {
+            trigger_error('Please extend this class, it will be made abstract in 5.1.', E_USER_DEPRECATED);
```
Comment (Devolicious) [inline src/Kunstmaan/ArticleBundle/Entity/AbstractCategory.php]: idem dito previous comment

File: src/Kunstmaan/ArticleBundle/Entity/AbstractTag.php [hunk 3]
```diff
@@ -10,6 +10,16 @@
 
 class AbstractTag extends AbstractEntity implements Translatable
 {
+    /**
+     * AbstractTag constructor.
+     */
+    public function __construct()
+    {
+        if (get_class($this) === AbstractTag::class) {
+            trigger_error('Please extend this class, it will be made abstract in 5.1.', E_USER_DEPRECATED);
```
Comment (Devolicious) [inline src/Kunstmaan/ArticleBundle/Entity/AbstractTag.php]: idem dito previous comment

**Comment by treeleaf** [comment]
Changes have been done.

**Comment by sandergo90** [comment]
I thinks this was not the issue in #1760. The classes are having a name "Abstract.." now, but the Abstract should be deprecated. Your deprecation warning says that it will be made abstract. But there is no need for it.

**Pull request merged**